main.py

```diff
@@ -64,31 +63,41 @@ def main():
             "WARNING: --limit SHOULD ONLY BE USED FOR TESTING. REAL METRICS SHOULD NOT BE COMPUTED USING LIMIT."
         )
 
-    if args.tasks is None:
-        task_names = tasks.ALL_TASKS
-    else:
-        task_names = pattern_match(args.tasks.split(","), tasks.ALL_TASKS)
-
-    print(f"Selected Tasks: {task_names}")
-
     description_dict = {}
     if args.description_dict_path:
         with open(args.description_dict_path, "r") as f:
             description_dict = json.load(f)
 
+    model = "hf"
+    uuid = hparams["model_args"]["uuid"]
+    if uuid is None:
+        model_args = (
+            f'pretrained={hparams["model_args"]["pretrained_model_name_or_path"]}'
+        )
+    else:
+        model_args = None
+
+    # GG_NOTE: task will always be a single string, but it may be a glob-pattern which will get
+    # converted to multiple tests.
+    assert isinstance(hparams["task"], str)
+    task_names = pattern_match([hparams["task"]], tasks.ALL_TASKS)
```

Comment:
What do you mean by searchable? Agree that having the mix of CLI and hp dicts is bad

Reply:
I meant searchable via the Determined searcher -- something needs to be a Determined `hyperparameter` before you can perform a hyperparameter search over it.

Reply:
Ah cool, got it.  In case someone is interested in searching over `num_fewshot` or something?

If it's alright by you, I'd like to leave that to a future PR.

Reply:
Yep, and sounds good to me.